subscription/index.ts

```diff
@@ -85,13 +87,15 @@ export const subscription = (<Data, Error>(useSWRNext: SWRHook) =>
         // Prevent frequent unsubscribe caused by unmount
         setTimeout(() => {
           // TODO: Throw error during development if count is undefined.
-          const count = subscriptions.get(subscriptionKey)! - 1
+          const count = subscriptions.get(subscriptionKey)
+          if (count == null) return
```

Comment:
This will never happen I believe?

Reply:
yeah for getting rid of the linting issue, but also was worrying about the setTimeout. 
And we have a comment: `// TODO: Throw error during development if count is undefined.`